### PR TITLE
8331 - Filed By

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: shared/src/business/entities/DocketEntry.test.js [hunk 1]
```diff
@@ -627,6 +710,614 @@ describe('DocketEntry entity', () => {
     });
   });
 
+  describe('generate filed by string', () => {
+    it('should generate correct filedBy string for single petitioner in filers', () => {
+      const docketEntry = new DocketEntry(
+        {
+          attachments: false,
```
Comment (rachaelparris) [inline shared/src/business/entities/DocketEntry.test.js]: To keep these tests cleaner, could you splat in a valid docket entry and only override the fields that are relevant to each test?

File: shared/src/business/entities/DocketEntry.test.js [hunk 2]
```diff
@@ -627,6 +710,614 @@ describe('DocketEntry entity', () => {
     });
   });
 
+  describe('generate filed by string', () => {
```
Comment (rachaelparris) [inline shared/src/business/entities/DocketEntry.test.js]: There is a separate file for these tests - DocketEntry.generateFiledBy.test.js

File: shared/src/business/useCaseHelper/consolidatedCases/getConsolidatedCasesForLeadCase.test.js [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
 const { MOCK_CASE } = require('../../../test/mockCase');
+// const { ROLES } = require('../../entities/EntityConstants');
 
 describe('getConsolidatedCasesForLeadCase', () => {
+  // beforeEach(() => {
+  //   applicationContext.getCurrentUser.mockReturnValue({
+  //     role: ROLES.docketClerk,
+  //     userId: faker.datatype.uuid(),
+  //   });
+  // });
```
Comment (rachaelparris) [inline shared/src/business/useCaseHelper/consolidatedCases/getConsolidatedCasesForLeadCase.test.js]: Remove

File: shared/src/business/entities/DocketEntry.test.js (hunk 2 above)
Comment (sutt0n) [reply shared/src/business/entities/DocketEntry.test.js]: Yeah, this was due to a botched merge conflict on my end. Good catch :)

**Comment by sutt0n** [comment]
Note: ad083a8 addresses DeepScan errors https://deepscan.io/dashboard/#view=project&tid=8976&pid=17137&bid=383813&subview=pull-request&prid=778091

**Review by sutt0n — APPROVED** [review]

**Pull request merged**